crates/client/src/rr/dnssec/tsig.rs

```diff
@@ -173,7 +175,7 @@ impl MessageFinalizer for TSigner {
         message: &Message,
         current_time: u32,
     ) -> ProtoResult<(Vec<Record>, Option<MessageVerifier>)> {
-        log::debug!("signing message: {:?}", message);
+        debug!("signing message: {:?}", message);
```

Comment:
could be a structured field
```suggestion
        debug!(?message, "signing message");
```